### PR TITLE
Prevent import of MUI `Tooltip`

### PR DESCRIPTION
**Pull request by juliawegmayr**
## Description

Prevent imports of `Tooltip` from `@mui/material`

## Changeset

-   [x] I have verified if my change requires a changeset

## Related tasks and documents

(https://vivid-planet.atlassian.net/browse/COM-1263)

### CODE REVIEW

**Review by johnnyomair — COMMENTED** [review]
This could fixed automatically in our projects by creating an upgrade script. Are you up for a challenge?

**Review by johnnyomair — CHANGES_REQUESTED** [review]
Please add a section to the upgrade guide.

**Comment by juliawegmayr** [comment]
> Please add a section to the upgrade guide.

Added here: 24cd93df921972878483c4c806de7686839d0d87

File: docs/docs/migration/migration-from-v7-to-v8.md [hunk 1]
```diff
@@ -172,6 +172,13 @@ Perform the following changes:
     }
     ```
 
+### Import `Tooltip` from `@comet/admin` package
+
+````diff
```
[the remark this answers is not in the document]
Comment (juliawegmayr) [reply docs/docs/migration/migration-from-v7-to-v8.md]: 4fae84c1439243cabe1c59022497963a5b71b834

**Review by johnnyomair — DISMISSED** [review]

**Comment by johnnyomair** [comment]
@juliawegmayr please resolve conflicts.

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**